src/sys/memfd.rs

```diff
@@ -29,6 +29,49 @@ libc_bitflags!(
         ///
         /// [`memfd_create(2)`]: https://man7.org/linux/man-pages/man2/memfd_create.2.html
         MFD_ALLOW_SEALING;
+        /// Anonymous file will be created using huge pages. It should be safe now to
+        /// combine with [`MFD_ALLOW_SEALING`] too.
+        /// However, despite its presence, on FreeBSD it is unimplemented for now (ENOSYS).
+        ///
+        /// See also the hugetlb filesystem in [`memfd_create(2)`].
+        ///
+        /// [`memfd_create(2)`]: https://man7.org/linux/man-pages/man2/memfd_create.2.html
+        #[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
```suggestion
        #[cfg(linux_android)]
```

Please use the alias instead